deepchem/molnet/load_function/zinc15_datasets.py

```diff
@@ -0,0 +1,252 @@
+    transformer_kwargs: Dict[str, Dict[str, object]] = {
+        'NormalizationTransformer': {
+            'transform_X': True
+        }
+    },
+    dataset_size: str = '250K',
+    dataset_dimension: str = '2D',
+    test_run: bool = False,
+    **kwargs) -> Tuple[List, Optional[Tuple], List]:
+  """Load zinc15.
+
+  ZINC15 is a dataset of over 230 million purchasable compounds for
+  virtual screening of small molecules to identify structures that
+  are likely to bind to drug targets. ZINC15 data is currently available
+  in 2D (SMILES string) format.
+
+  MolNet provides subsets of 250K, 1M, and 10M "lead-like" compounds
+  from ZINC15. The full dataset of 270M "goldilocks" compounds is also
+  available. Compounds in ZINC15 are labeled by their molecular weight 
+  and LogP (solubility) values. Each compound also has information about how
+  readily available (purchasable) it is and its reactivity. Lead-like
+  compounds have molecular weight between 300 and 350 Daltons and LogP
+  between -1 and 3.5. Goldilocks compounds are lead-like compounds with
+  LogP values further restricted to between 2 and 3.
+
+  If `reload = True` and `data_dir` (`save_dir`) is specified, the loader
+  will attempt to load the raw dataset (featurized dataset) from disk.
+  Otherwise, the dataset will be downloaded from the DeepChem AWS bucket.
+
+  For more information on ZINC15, please see [1]_ and
+  https://zinc15.docking.org/.
+
+  Parameters
+  ----------
+  size : str (default '250K')
+    Size of dataset to download. Currently only '250K' is supported.
+  format : str (default '2D')
+    Format of data to download. 2D SMILES strings or 3D SDF files.
+  featurizer : allowed featurizers for this dataset
+    A featurizer that inherits from deepchem.feat.Featurizer.
+  transformers : List of allowed transformers for this dataset
+    A transformer that inherits from deepchem.trans.Transformer.
+  splitter : allowed splitters for this dataset
+    A splitter that inherits from deepchem.splits.splitters.Splitter.
+  reload : bool (default True)
+    Try to reload dataset from disk if already downloaded. Save to disk
+    after featurizing.
+  data_dir : str, optional (default None)
+    Path to datasets.
+  save_dir : str, optional (default None)
+    Path to featurized datasets.
+  featurizer_kwargs : dict
+    Specify parameters to featurizer, e.g. {"size": 1024}
+  splitter_kwargs : dict
+    Specify parameters to splitter, e.g. {"seed": 42}
+  transformer_kwargs : dict
+    Maps transformer names to constructor arguments, e.g.
+    {"BalancingTransformer": {"transform_x":True, "transform_y":False}}
+  dataset_size : str (default '250K')
+    Number of compounds to download; '250K', '1M', '10M', or '270M'.
+  dataset_dimension : str (default '2D')
+    SMILES strings (2D) or 3D SDF files; '2D' or '3D'
+  test_run : bool (default False)
+    Flag to indicate tests, if True dataset is not downloaded.
+  **kwargs : additional optional arguments.
```

Comment:
Are there any additional `kwargs` we use? If not, perhaps we should add a note here or even remove `**kwargs` from the function

Reply:
No need for `kwargs`, deleted!